src/admin-toolkit-ui/index.tsx

```diff
@@ -420,12 +270,12 @@ const uiComponent = (
   sdkHelpers?: ISDKHelpers,
   playersHelper?: IPlayersHelper,
 ) => {
-  const adminToolkitEntitie = getAdminToolkitComponent(engine)
+  const adminToolkitEntity = getAdminToolkitComponent(engine)
   const player = playersHelper?.getPlayer()
-  const isPlayerAdmin = isAllowedAdmin(engine, adminToolkitEntitie, player)
-  const scaleFactor = getScaleUIFactor(engine)
+  const isPlayerAdmin = isAllowedAdmin(engine, adminToolkitEntity, player)
```

Comment:
`adminToolkitEntity` not used by the method `isAllowedAdmin`
```suggestion
  const isPlayerAdmin = isAllowedAdmin(engine, player)
```